docs/.vitepress/config.mts

```diff
@@ -103,6 +103,7 @@ export default defineConfig({
         items: [
           { text: "What is Melange", link: "/what-is-melange" },
           { text: "Rationale", link: "/rationale" },
+          { text: "Syntaxes", link: "/syntaxes" },
```

Comment:
It's in its own page, but maybe it should be merged into Getting Started instead?

Reply:
For some reason "Syntaxes" doesn't sound good to me (?), maybe it's just an impression but would "Supported syntaxes" make more sense or I'm tripping?

Reply:
I would put something about the syntaxes in the 'getting started' section as the first thing (almost like a warning, and how to toggle the syntax on the site), because the first sentence under 'getting started' is "If you would like to learn OCaml and Melange from scratch, we recommend you to read Melange for React Devs", but that site is all in reason.

Reply:
Renamed to "Supported syntaxes" in https://github.com/melange-re/melange-re.github.io/pull/200/commits/5bacbf32a9df043bb0eb2a2fc561b9c20a0f0eba